CONTRIBUTING.md

```diff
@@ -41,9 +41,14 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests!
 
-1. Fork the repo and create your branch from `master`.
+If you need help with GIT or our workflow, please ask on [Gitter.im](https://gitter.im/netlify/NetlifyCMS). We want your contributions even if you're just learning GIT. Our maintainers are happy to help!
```

Comment:
"git" - lowercase it unles it's the first word in the sentence. Closing thing I could find to a canonical reference: http://git.661346.n2.nabble.com/Consensus-on-quot-Git-quot-td3984988.html.

Reply:
I use https://git-scm.com/ as the canonical reference. They spell it "Git."

Reply:
(and I do prefer it lowercase! but hey, I defer to the official brand—and blatantly ignore sentence capitalization in this comment in protest 😛 )

Reply:
That's a better reference, agreed.